src/frontend/views/invoice.njk

```diff
@@ -106,7 +111,9 @@
       </tbody>
     {% endfor %}
   </table>
-  <div class="govuk-!-padding-bottom-7"></div>
+  <p class="govuk-body-s govuk-!-padding-bottom-7">
+    <span class="govuk-!-font-weight-bold">Note:</span> If there is no invoice price because invoice data is missing, "Invoice data missing" will be displayed. If there is no measured price because events are missing, display "Events missing" will be displayed. In either situation occurs, the price discrepancy column will be blank.
```

Comment:
```suggestion
    <span class="govuk-!-font-weight-bold">Note:</span> If there is no invoice price because invoice data is missing, "Invoice data missing" will be displayed. If there is no measured price because events are missing, "Events missing" will be displayed. In either situation occurs, the price discrepancy column will be blank.
```